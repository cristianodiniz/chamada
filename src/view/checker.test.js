import React from "react";
import Checker from "./checker";
import renderer from "react-test-renderer";
import ReactDOM from "react-dom";

describe("Checker Tests Component", function() {
  it("Should render without crashing", async done => {
    const div = document.createElement("div");
    ReactDOM.render(<Checker />, div);
    ReactDOM.unmountComponentAtNode(div);
    done();
  }, 10000);

  test("Should Match Snapshot", async done => {
    const component = renderer.create(<Checker />);

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
    done();
  }, 10000);
});
