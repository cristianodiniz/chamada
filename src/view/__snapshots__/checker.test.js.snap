// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checker Tests Component Should Match Snapshot 1`] = `
<span
  className="Checker-container-61"
>
  <span
    className="MuiTypography-root Checker-inside-62 MuiTypography-body2 MuiTypography-colorTextPrimary"
  />
  <span
    className="MuiSwitch-root MuiSwitch-edgeEnd"
  >
    <span
      aria-disabled={false}
      className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-105 MuiSwitch-switchBase MuiSwitch-colorSecondary"
      onBlur={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={null}
    >
      <span
        className="MuiIconButton-label"
      >
        <span
          className="MuiSwitch-thumb"
        />
        <input
          aria-labelledby="switch-list-label-wifi"
          className="PrivateSwitchBase-input-108 MuiSwitch-input"
          onChange={[Function]}
          type="checkbox"
        />
      </span>
    </span>
    <span
      className="MuiSwitch-track"
    />
  </span>
</span>
`;
